File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 4 - C/q14.c
//Escreva um algoritmo que, para uma conta bancária, leia o seu número, o saldo, o tipo de operação
//a ser realizada (depósito ou retirada) e o valor da operação. Após, determine e mostre o novo saldo. Se o novo
//saldo ficar negativo, deve ser mostrada, também, a mensagem “conta estourada”.

#include<stdio.h>
int main(){
	int n;
	float saldo, valor_op, saldo_novo;
	char op;
	printf("Insira o numero da conta: ");
	scanf("%d", &n);
	printf("Insira o valor do saldo: R$");
	scanf("%f", &saldo);
	printf("Tipo de operação, digite D(depósito) ou R(retirada): ");
	scanf("%s", &op);
	printf("Valor da operacao: ");
	scanf("%f", &valor_op);
	if(op=='D' || op=='d'){
		saldo_novo=saldo+valor_op;
	    printf("Novo saldo: R$ %f", saldo_novo);
	}   
	else if(op=='R' || op=='r'){
		saldo_novo=saldo-valor_op;
	    printf("Novo saldo: R$ %f", saldo_novo);
	}
	if(saldo_novo<=0){
		printf("Conta estourada");
	}
	        
}

File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 4 - Python/q14.py
#Escreva um algoritmo que, para uma conta bancária, leia o seu número, o saldo, o tipo de operação
#a ser realizada (depósito ou retirada) e o valor da operação. Após, determine e mostre o novo saldo. Se o novo
#saldo ficar negativo, deve ser mostrada, também, a mensagem “conta estourada”.

n=int(input("Insira o número da conta: "))
saldo=float(input("Insira o valor do saldo: R$"))
op=str(input("Tipo de operação, digite D(depósito) ou R(retirada): "))
valor_op=float(input("Valor da operação: R$"))

if op=="D" or op=="d":
    saldo_novo=saldo+valor_op
    print("Novo saldo: R$", saldo_novo)
elif op=="R" or op=="r":
    saldo_novo=saldo-valor_op
    print("Novo saldo: R$", saldo_novo)
    if saldo_novo<=0:
        print("Conta estourada")
    
    
